README.md

```diff
@@ -1 +1,45 @@
-# Hello-Microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> Linters installation.
+
+
+## Built With
+
+- HTML
+- CSS
+- Linters
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
```

Comment:
- [ ] It would be better to modify the `Getting started` section and add some steps on how the users can use your project locally.